pom.xml

```diff
@@ -100,6 +103,7 @@
 			<layout>p2</layout>
 			<url>${jbosstools-central-site}</url>
 		</repository>
+		<!-- End concerned by comment FUSETOOLS-2547 -->
```

Comment:
I'd replace with "End of repository definitions causing a warning (see FUSETOOLS-2547)"